### PR TITLE
Signup form modified to contain first_name, last_name and mobile number

### PR DESCRIPTION
**Pull request by DESHRAJ**
This pull request resolves the issue #88 . @vnbang2003 Please let me know if any changes are required.

### CODE REVIEW

**Comment by DESHRAJ** [comment]
Ok I can see that build is not passing because I need update the signup_flow test. I will do this by today itself. :)

File: wye/profiles/forms.py [hunk 1]
```diff
@@ -10,24 +11,20 @@ class UserAuthenticationForm(AuthenticationForm):
...
+    class Meta:
+        model = get_user_model()
+        fields = ['first_name', 'last_name']
+        widgets = {
+            'first_name': forms.TextInput(attrs={'placeholder': 'First Name'}),
+            'last_name': forms.TextInput(attrs={'placeholder': 'Last Name'}),
+        }
+    def save(self, user):
+        user.save()
+        models.Profile.objects.filter(user__id = user.id).update(mobile = self.cleaned_data['mobile'])
```
Comment (ultimatecoder) [inline wye/profiles/forms.py]: @DESHRAJ : No need to increase query traffic by filter Profile object again. **user** is already related with **profile** on one to one bases. I think assignment of **mobile** should be done with following approach
```python
user.profile.mobile = self.cleaned_data['mobile']
user.save()
```

**Comment by DESHRAJ** [comment]
Thanks a lot @ultimatecoder for helping. I will fix this and update the test.

**Comment by DESHRAJ** [comment]
This PR not passing the build tests. @vnbang2003 can you help me with this ?

**Comment by vnbang2003** [comment]
@DESHRAJ Below is error i see  with flake8 standard. https://travis-ci.org/pythonindia/wye/builds/88378616

0.80s$ flake8

./wye/profiles/forms.py:2:1: F401 'UserCreationForm' imported but unused

./wye/profiles/forms.py:14:1: E302 expected 2 blank lines, found 1

./wye/profiles/forms.py:16:33: E127 continuation line over-indented for visual indent

./wye/profiles/forms.py:17:33: E127 continuation line over-indented for visual indent

./wye/profiles/forms.py:18:5: E301 expected 1 blank line, found 0

./wye/profiles/forms.py:25:5: E301 expected 1 blank line, found 0

./wye/profiles/forms.py:29:1: E302 expected 2 blank lines, found 1

./wye/profiles/forms.py:37:1: W391 blank line at end of file

File: wye/profiles/forms.py [hunk 2]
```diff
@@ -10,24 +11,20 @@ class UserAuthenticationForm(AuthenticationForm):
...
+                                widget=forms.TextInput(attrs={'placeholder': 'Mobile Number'}))
+    class Meta:
+        model = get_user_model()
+        fields = ['first_name', 'last_name']
+        widgets = {
+            'first_name': forms.TextInput(attrs={'placeholder': 'First Name'}),
+            'last_name': forms.TextInput(attrs={'placeholder': 'Last Name'}),
+        }
+    def save(self, user):
+        user.profile.mobile = self.cleaned_data['mobile']
```
Comment (ultimatecoder) [inline wye/profiles/forms.py]: Well, I forgot to tell that, after assigning **user.profile.mobile** **user.profile.save()** will save the actual profile modifications. You can find any other way to save the profile record update. Thanks!

**Comment by ultimatecoder** [comment]
@DESHRAJ As @vnbang2003  has mentioned validation for mobile number, I am not able to see the validation for it in this code. I think it will be good to add number only and length validation for mobile number field with proper error message. Thanks!

**Pull request merged**